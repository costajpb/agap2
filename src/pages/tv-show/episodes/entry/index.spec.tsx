import { render } from "@testing-library/react"
import Entry from "."

describe('pages/tv-show/episodes/entry', () => {
    it('should render', () => {
        const episode = {
            id: 1,
            title: 'Dummy title'
        }

        const { container } = render(<Entry episode={episode} />)

        expect(container).toMatchSnapshot()
    })
})