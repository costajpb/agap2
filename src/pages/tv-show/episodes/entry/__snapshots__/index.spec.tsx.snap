// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pages/tv-show/episodes/entry should render 1`] = `
<div>
  <li
    class="sc-gmPhUn jdersu"
  >
    <a
      class="sc-aXZVg hEUmpn"
      data-episode-id="1"
      data-testid="episode"
      href="/episodes/1"
    >
      <img
        alt="Dummy title"
        class="sc-fBWQRz jOjvwV"
      />
      <span
        class="sc-kMkxaj gmRHkp"
      >
        Dummy title
      </span>
    </a>
  </li>
</div>
`;
